web-app/src/app/router/Router.tsx

```diff
@@ -23,6 +24,7 @@ export const AppRouter: React.FC = () => {
         <Route path='account' element={<Account />} />
       </Route>
       <Route path='contact-info' element={<ContactInformation />} />
+      <Route path='change-password' element={<ChangePassword />} />
```

Comment:
This should be a protected screen.
```suggestion
<Route element={<ProtectedRoute />}>
      <Route path='change-password' element={<ChangePassword />} />
</Route>
```